package common

func HandleErr(err error) {
  if err != nil {
    panic(err2)
  }
}
